Sources/Settings/SettingsWindowController.swift

```diff
@@ -97,7 +97,15 @@ public final class SettingsWindowController: NSWindowController {
 
 		showWindow(self)
 		restoreWindowPosition()
-		NSApp.activate(ignoringOtherApps: true)
+		#if compiler(>=5.9) && canImport(AppKit, _version: "14.0")
```

Comment:
```suggestion
		#if compiler(>=5.9) && canImport(AppKit)
```

I don't think we need to check the AppKit version as the availability check already does that. And it's also marked as internal so I wouldn't want to rely on it.